.gitignore

```diff
@@ -52,8 +52,8 @@ coverage.xml
 .pytest_cache/
 
 # Translations
-*.mo
-*.pot
+#*.mo
```

Comment:
Without the *.mo everyone needs to compile their own translation files, right?
I think we should include all files necessary to run the code out of the box.

Reply:
Or should we as the repo maintainers upload changes manually? Perhaps just a bit confused by the unfamiliar file extensions :)